src/main/c/openssl/extconf.rb

```diff
@@ -30,11 +30,10 @@
 
 if ENV['OPENSSL_PREFIX']
   $CFLAGS += " -I #{ENV['OPENSSL_PREFIX']}/include"
-  $LIBS += " -l #{ENV['OPENSSL_PREFIX']}/lib/libssl.#{RbConfig::CONFIG['NATIVE_DLEXT']}"
-else
-  $LIBS += " -l libssl.#{RbConfig::CONFIG['NATIVE_DLEXT']}"
 end
 
+$LIBS += " -l libssl.#{RbConfig::CONFIG['NATIVE_DLEXT']}"
```

Comment:
That's an `autoconf` option - we don't use autoconf in our system.

I'm not sure about combining finding the includes with finding the library. What about making the first thing more precise - make it `OPENSSL_INCLUDE_DIR`?